mne/viz/_brain/_timeviewer.py

```diff
@@ -225,90 +255,105 @@ def __init__(self, brain):
 
         # time slider
         max_time = len(brain._data['time']) - 1
+        self.time_call = SmartSlider(
+            plotter=self.plotter,
+            callback=self.brain.set_time_point,
+            name="time"
+        )
         time_slider = self.plotter.add_slider_widget(
-            brain.set_time_point,
+            self.time_call,
             value=brain._data['time_idx'],
             rng=[0, max_time],
             pointa=(0.23, 0.1),
             pointb=(0.77, 0.1),
             event_type='always'
         )
-        time_slider.name = "time_slider"
+        time_slider.name = "time"
 
         # playback speed
         default_playback_speed = 0.05
+        self.playback_speed_call = SmartSlider(
+            plotter=self.plotter,
+            callback=self.set_playback_speed,
+            name="playback_speed"
+        )
         playback_speed_slider = self.plotter.add_slider_widget(
-            self.set_playback_speed,
+            self.playback_speed_call,
             value=default_playback_speed,
-            rng=[0.01, 1], title="playback speed",
+            rng=[0.01, 1], title="play speed",
```

Comment:
We should just use `speed`, it's short but still descriptive. Let's do it in another PR unless you're planning to push more commits

Reply:
let's do it here. it's a very short change